stream-compositions/api/product-catalog-stream-compositions-api/product-catalog-stream-composition-service-api-v1.yaml

```diff
@@ -0,0 +1,18 @@
+openapi: 3.0.3
```

Comment:
Do we need service api for composition ? isn't client api enough ? - same questions for other composition types